### PR TITLE
This is not a real PR, just to be able to comment on all the lines

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: step-00/lib/main.dart [hunk 1]
```diff
@@ -1,24 +1 @@
...
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-import 'package:flutter/cupertino.dart';
-import 'package:flutter/services.dart';
-import 'app.dart';
-
-void main() {
-  SystemChrome.setPreferredOrientations(
```
Comment (xster) [inline step-00/lib/main.dart:20]: Would this need accompanying narratives to explain what this is for?

File: step-02/lib/product_row_item.dart [hunk 2]
```diff
@@ -1,110 +1 @@
...
-      children: <Widget>[
-        row,
-        Padding(
-          padding: const EdgeInsets.only(
-            left: 100,
-            right: 16,
-          ),
-          child: Container(
-            height: 1,
-            color: const Color(0xFFD9D9D9),
```
Comment (xster) [inline step-02/lib/product_row_item.dart:104]: Maybe just reference the styles file you already made

File: step-03/lib/search_bar.dart [hunk 3]
```diff
@@ -1,56 +1 @@
...
-            ),
-            Expanded(
-              child: CupertinoTextField(
-                controller: controller,
-                focusNode: focusNode,
-                style: Styles.searchText,
-                cursorColor: Styles.searchCursorColor,
-              ),
-            ),
-            GestureDetector(
```
Comment (xster) [inline step-03/lib/search_bar.dart:44]: I forget what we said in the other commit. Could we just use the clear button in the CupertinoTextField?

File: step-03/lib/search_tab.dart [hunk 4]
```diff
@@ -1,103 +1 @@
...
-          decoration: const BoxDecoration(
-            color: Styles.scaffoldBackground,
-          ),
-          child: SafeArea(
-            child: Column(
-              children: [
-                _createSearchBox(),
-                Expanded(
-                  child: ListView(
-                    children: _generateProductRows(model.search(_terms)),
```
Comment (xster) [inline step-03/lib/search_tab.dart:93]: If this whole thing is wrapped in a safe area, I suspect you won't be able to scroll search result items underneath the bottom tab bar.

File: step-06/lib/shopping_cart_tab.dart [hunk 5]
```diff
@@ -1,334 +1 @@
...
-              children: const <Widget>[
-                Icon(
-                  CupertinoIcons.clock,
-                  color: CupertinoColors.lightBackgroundGray,
-                  size: 28,
-                ),
-                SizedBox(width: 6),
-                Text(
-                  'Delivery time',
-                  style: TextStyle(
```
Comment (xster) [inline step-06/lib/shopping_cart_tab.dart:129]: Be consistent with declaring styles in the styles file vs inline

**Review by xster — COMMENTED** [review]
Cool, not many comments, just a few minor things.

File: step-03/lib/search_bar.dart (hunk 3 above)
Comment (domesticmouse) [reply step-03/lib/search_bar.dart:44]: The clear button in CupertinoTextField doesn't work with the dark background this search text field has. This code is lifted from @redbrogdon's Veggie Seasons sample, so maybe he can give more colour.

File: step-00/lib/main.dart (hunk 1 above)
Comment (domesticmouse) [reply step-00/lib/main.dart:20]: Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-babe824401a3594961b3ce1a44c1baa9

File: step-02/lib/product_row_item.dart (hunk 2 above)
Comment (domesticmouse) [reply step-02/lib/product_row_item.dart:104]: Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-e984079777bf36788a762b77ed2c1615

File: step-03/lib/search_tab.dart (hunk 4 above)
Comment (domesticmouse) [reply step-03/lib/search_tab.dart:93]: I'm not seeing that behaviour in the simulator. Maybe clone the repo and run it to check?

File: step-06/lib/shopping_cart_tab.dart (hunk 5 above)
Comment (domesticmouse) [reply step-06/lib/shopping_cart_tab.dart:129]: Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-08987380babd0f4baf7e95a853929d94

File: step-03/lib/search_bar.dart (hunk 3 above)
Comment (xster) [reply step-03/lib/search_bar.dart:44]: Doh! Can you open an issue on the main repo?

Comment (domesticmouse) [reply step-03/lib/search_bar.dart:44]: Whups, sorry I missed this. I see you have already pushed through a PR for this. Any ideas on when it will land?

**Pull request closed without merging**